src/codegen/sys/lib_.rs

```diff
@@ -294,18 +300,40 @@ fn generate_enums(w: &mut dyn Write, env: &Env, items: &[&Enumeration]) -> Resul
             let version = member_config.iter().filter_map(|m| m.version).next();
 
             if is_alias {
-                continue;
+                return quote! {};
             }
 
-            version_condition(w, env, version, false, 0)?;
-            writeln!(
-                w,
-                "pub const {}: {} = {};",
-                member.c_identifier, item.c_type, member.value,
-            )?;
+            let c_identifier = Ident::new(&member.c_identifier, Span::call_site());
+            let value = Literal::i32_unsuffixed(member.value.parse().unwrap());
```

Comment:
IMHO we get problems with `i32`

Reply:
Why / when?

Reply:
I not remember where, but there was constant that not fill into i32, only in u32.
But it really not problem as we declare type as c_int.
Sorry, my mistake